README.md

```diff
@@ -119,17 +119,6 @@ Or to just see HTTP traffic:
 java -jar jenkins-cli.jar -s http://localhost:8080/jenkins/ tail-log org.jclouds.rest.internal.InvokeHttpMethod -l FINE
 ```
 
-# Running on Agents outside AWS
-
-Create 2 agents in Jenkins, one for uploading and another for downloading, and launch them with Docker
-
-```sh
-docker run --rm -i --init -v <YOUR_HOME_DIR>/.aws:/home/jenkins/.aws -e AWS_REGION=… -e AWS_PROFILE=… jenkins/slave:alpine java -jar /usr/share/jenkins/slave.jar
```

Comment:
This section has been wrong for a while now. You need not, and should not, mount `~/.aws/` or set any special environment variables (since we started using presigned URLs for all operations on the agent)…meaning that there are really no special instructions at all, and the section can simply be deleted.